### DISCUSSION TITLE
Is it possible to store the values in Sharepoint column which is updated by user while generating the template in Microsoft Syntax

### DISCUSSION THREAD

**Discussion by spo365** [pnp/syntex-samples#27] [Q&A] [answered]
[](url)Hi Syntex expect ,

I am trying to save values in Sharepoint libraries when user save any values in the template which I have created for generating the document.
Refer the below image circle in the red.
Can we store these fields in SharePoint as we need to generate more document by using these values ?
![image](https://user-images.githubusercontent.com/5062256/212821135-05013775-5b33-4b0a-9248-e7526319b3ce.png)

thanks in Advance

**Comment by Spucelik** [comment, marked as answer]
When you're creating a document using manual input fields, the best way to capture those values is to have a model run against the newly created document and extract the manually entered values.  If the data in the field is derived from another list/library, you can always use that source for further business processing.
